Migrated Buckets Pages to mds

bexsoft opened the pull request:
## What does this do?

- Migrated the Buckets page to the available mds components
- Updated mds to v0.8.2

## How does it look?

<img width="602" alt="Screenshot 2023-07-24 at 21 15 00" src="https://github.com/minio/console/assets/33497058/ae7470d2-1bde-4156-b2d2-e6c5c4341680">
<img width="876" alt="Screenshot 2023-07-24 at 21 14 54" src="https://github.com/minio/console/assets/33497058/54c7f9a2-0aac-49bd-8d23-e5f7a5a49d50">
<img width="846" alt="Screenshot 2023-07-24 at 21 14 40" src="https://github.com/minio/console/assets/33497058/01f427b1-42f9-46f5-b793-d45f66118cfe">
<img width="979" alt="Screenshot 2023-07-24 at 21 14 35" src="https://github.com/minio/console/assets/33497058/f2d5f8d9-313f-4ffa-9a49-addf8c64686f">
<img width="610" alt="Screenshot 2023-07-24 at 21 14 27" src="https://github.com/minio/console/assets/33497058/8f65a622-2968-4141-8712-fe2e1c34852d">
<img width="1091" alt="Screenshot 2023-07-24 at 21 14 22" src="https://github.com/minio/console/assets/33497058/2db5f9b5-c574-4f18-b09f-d92c7ef740c0">
<img width="1686" alt="Screenshot 2023-07-24 at 21 14 19" src="https://github.com/minio/console/assets/33497058/9c1979f1-04d6-4e6b-a097-c01a74cb543e">
<img width="1681" alt="Screenshot 2023-07-24 at 21 14 15" src="https://github.com/minio/console/assets/33497058/01453a50-e975-4e3a-b964-6b4ab7d724b2">
<img width="1675" alt="Screenshot 2023-07-24 at 21 14 10" src="https://github.com/minio/console/assets/33497058/470bda5a-7efc-40df-84ee-0320a9d32cb0">
<img width="1696" alt="Screenshot 2023-07-24 at 21 14 06" src="https://github.com/minio/console/assets/33497058/96cbf3d8-9bc0-4415-9820-49e9ecac69c6">
<img width="1680" alt="Screenshot 2023-07-24 at 21 14 01" src="https://github.com/minio/console/assets/33497058/7156ec05-3771-41ba-a154-c72746c1205d">
<img width="1693" alt="Screenshot 2023-07-24 at 21 13 55" src="https://github.com/minio/console/assets/33497058/2cbce9e8-8f11-4084-8e75-4ccb1b85c2a6">
<img width="1619" alt="Screenshot 2023-07-24 at 21 13 50" src="https://github.com/minio/console/assets/33497058/f8c33266-b6cf-4461-8213-3626566574a3">
<img width="1694" alt="Screenshot 2023-07-24 at 21 13 45" src="https://github.com/minio/console/assets/33497058/3eb80075-2c39-49ff-b57c-9d33b3a73b56">

prakashsvmx commented:
@bexsoft 

Changes look good to me :+1: 

Some Observations:

Adding bucket tag and removing does not reflect if there is only one tag. ( after deleting, page reload is required. "Refresh" also does not reflect)

<details>
<summary>
When creating a bucket with Object Locking and setting retention, the toggle could have green color as it is on. the current disabled state is makes it unclear
</summary>

![image](https://github.com/minio/console/assets/23444145/33dec85c-6e35-4013-8af6-ab34571b98fb)

</details>

oscarocastellanos reviewed (dismissed):
LGTM

bexsoft commented:
> @bexsoft
> 
> Changes look good to me 👍
> 
> Some Observations:
> 
> Adding bucket tag and removing does not reflect if there is only one tag. ( after deleting, page reload is required. "Refresh" also does not reflect)
> 
> When creating a bucket with Object Locking and setting retention, the toggle could have green color as it is on. the current disabled state is makes it unclear

Addressed comments
